node/index.js

```diff
@@ -1,26 +1,29 @@
-'use strict';
+// read env vars from .env file
+require("dotenv").config();
 
-var util = require('util');
+const util = require("util");
+const express = require("express");
+const bodyParser = require("body-parser");
+const moment = require("moment");
+const plaid = require("plaid");
 
-var envvar = require('envvar');
-var express = require('express');
-var bodyParser = require('body-parser');
-var moment = require('moment');
-var plaid = require('plaid');
-
-var APP_PORT = envvar.number('APP_PORT', 8000);
-var PLAID_CLIENT_ID = envvar.string('PLAID_CLIENT_ID');
-var PLAID_SECRET = envvar.string('PLAID_SECRET');
-var PLAID_ENV = envvar.string('PLAID_ENV', 'sandbox');
+const APP_PORT = process.env.APP_PORT || 8000;
+const PLAID_CLIENT_ID = process.env.PLAID_CLIENT_ID;
+const PLAID_SECRET = process.env.PLAID_SECRET;
+const PLAID_ENV = process.env.PLAID_ENV || "sandbox";
```

Comment:
everything below this point is just prettier formatting and cost/let/var changes

Reply:
I thought we reco'd single quotes for prettier?

Reply:
Probably makes sense to commit a `.prettierrc` here if we are going to run it.